README.md

```diff
@@ -1,110 +1,94 @@
-# ASP.NET Core Feature Flags
+# .NET Feature Management
 
-Feature flags provide a way for ASP.NET Core applications to turn features on or off dynamically. Developers can use feature flags in simple use cases like conditional statements to more advanced scenarios like conditionally adding routes or MVC filters. Feature flags build on top of the .NET Core configuration system. Any .NET Core configuration provider is capable of acting as the back-bone for feature flags.
+The Microsoft.FeatureManagement library enables developers to use feature flags and dynamic features inside of their applications. Feature flags can be used to turn features on or off dynamically. Developers can use feature flags in simple use cases like conditional statements to more advanced scenarios like conditionally adding routes or MVC filters. Dynamic features can be used to select different variants of a feature's configuration. This enables the possibility of using one version of a feature for one set of users, and another version of the feature for the remaining users.
 
 Here are some of the benefits of using this library:
 
 * A common convention for feature management
 * Low barrier-to-entry
   * Built on `IConfiguration`
   * Supports JSON file feature flag setup
-* Feature Flag lifetime management
+* Feature flag lifetime management
   * Configuration values can change in real-time, feature flags can be consistent across the entire request
-* Simple to Complex Scenarios Covered
+* Simple to complex scenarios covered
   * Toggle on/off features through declarative configuration file
-  * Dynamically evaluate state of feature based on call to server
+  * Use different variants of a feature in different circumstances
 * API extensions for ASP.NET Core and MVC framework
   * Routing
   * Filters
   * Action Attributes
 
 **API Reference**: https://go.microsoft.com/fwlink/?linkid=2091700
 
-### Feature Flags
-Feature flags are composed of two parts, a name and a list of feature-filters that are used to turn the feature on.
+## Index
+* [Feature Flags](./README.md#Feature-Flags)
+  * [Feature Flag Declaration](./README.md#Feature-Flag-Declaration)
+  * [Feature Filters](./README.md#Feature-Filters)
+  * [ASP.NET Core Integration](./README.md#ASPNET-Core-Integration)
+  * [Targeting](./README.md#Targeting)

```

Comment:
I don't see a section that is named "Targeting". Do you mean to point to the "Microsoft.Targeting" section?

BTW, to differentiate from the "Targeting assigner" below, can we name this "Targeting filter"?

Reply:
That's [this section](https://github.com/jimmyca15/FeatureManagement-Dotnet/blob/user/jimmyca/mReadme3/README.md#targeting). The section explains the concept of targeting. Since it's a bit of a generic explanation I moved it to after feature flags and dynamic features. Even though it does reference feature filters, it uses more of a tool in the explanation of targeting. So now, the targeting feature filter and the targeting feature variant assigner appear symmetric in the index, then the generic targeting explanation appears at a top level.